src/topics/models.py

```diff
@@ -49,13 +49,24 @@ def create_post(self, body: str, author_id: int) -> Post:
         return new_post
 
     @staticmethod
-    def get_topics(sorting: dict[str, str] | None = None) -> list[Topic]:
+    def get_topics(query_parameters: dict[str, list[Any]] | None = None) -> list[Topic]:
         """Use this method to get all topics from topics table."""
+        sorting = query_parameters["order_by"] if query_parameters else None
+        author_ids = query_parameters["author_id"] if query_parameters else None
+        created_after = query_parameters["created_after"] if query_parameters else None
+        created_before = query_parameters["created_before"] if query_parameters else None
```

Comment:
After we did a change from previous comment we can remove these lines:
```python
sorting = query_parameters["order_by"] if query_parameters else None
author_ids = query_parameters["author_id"] if query_parameters else None
created_after = query_parameters["created_after"] if query_parameters else None
created_before = query_parameters["created_before"] if query_parameters else None
```

because these lines in `routes.py` do exactly the same thing:
```python
topics = Topic.get_topics(
    sorting=parsed_args.get("order_by"),
    author_ids=parsed_args.get("author_ids"),
    created_before=parsed_args.get("created_before"),
    created_after=parsed_args.get("created_after"),
)
```